halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/OrcaProfileFactory.java

```diff
@@ -45,5 +45,7 @@ protected void setProfile(Profile profile, DeploymentConfiguration deploymentCon
     profile.appendContents("pipelineTemplates.enabled: " + pipelineTemplates);
     // For backward compatibility
     profile.appendContents("pipelineTemplate.enabled: " + pipelineTemplates);
+
+    profile.appendContents("global.spinnaker.timezone: " + deploymentConfiguration.getTimezone());
```

Comment:
Should this go in Orca's config or should this be inherited from the top level spinnaker.yml?

Reply:
Good call, moved to `SpinnakerProfileFactory`!